### PR TITLE
10_002.txt to 014_04.txt

### PR DESCRIPTION
**Pull request by Antielfi**
Another batch of file, those one should be ok without to much editing... I hope :D

### CODE REVIEW

File: Cornelica/scenario/013_03a.txt [hunk 1]
```diff
@@ -18,19 +18,19 @@
...
 
-[Justina] "Oh, if it isn't Lily, hehe, you're with a boy, are you on a date? "
+[Justina]"Oh, if it isn't Lily, hehe, you're with a boy, \nare you on a date?"
 
 @huki b=1 id=AC0
 @bs f=ユスティーナ@微笑 body=-1 face=0 x=350 y=50 t=100 wt=0
 @bs f=リリィ@焦り_r body=-1 face=0 x=-350 y=50 t=200
 
-[Lily] "Al isn't like that!"
+[Lily]"Al isn't like that!"
```
Comment (TwinDragon) [inline Cornelica/scenario/013_03a.txt]: This is just a copy/paste of the bit I suggested in Discord:

`ア、アルはそんなんじゃないよ！`

I see its put as `Al isn't like that!` - but I wonder if it'd make more sense as something like `No, it isn't like that!` maybe. But me, I'm just guessing. Google failed me again on that line somewhat

File: Cornelica/scenario/013_03a.txt [hunk 2]
```diff
@@ -90,7 +90,7 @@
 @bs f=ユスティーナ@笑み body=-1 face=0 x=350 y=50 t=100 wt=0
 @bs f=アル@困り body=-1 face=0 x=-350 y=50 t=200
 
-[Justina] "So you're Al, are you shopping too? Now if are, I'll have to add an additional charge of [extra] ... ♪" ♪
+[Justina]"So you're Al, are you shopping too? \nNow you if are, I'll have to add \nan additional charge of extra♪"
```
Comment (TwinDragon) [inline Cornelica/scenario/013_03a.txt]: maybe `Now if you are,` - lol machine translate fail

File: Cornelica/scenario/013_1.txt [hunk 3]
```diff
@@ -56,16 +56,16 @@
 
 @bs f=リリィ@微笑 x=350 y=50 t=400
 
-[Lily] "Moving onto " Magical power ", this is what is necessary for Al to move"
+[Lily]"Moving onto \C[2]Magical power\C[0], \nthis is what is necessary for you to move"
 
 @bs f=リリィ@哀しみ x=350 y=50 t=200
 
-[Lily] "Get anywhere near a Succubus, you'll suck up the Charm, and your magical power will decrease, so be careful."
+[Lily]"Get anywhere near a Succubus, you'll suck up the \C[2]Charm\C[0], \nand your \C[2]Magical power\C[0] will decrease, so be careful."
```
Comment (TwinDragon) [inline Cornelica/scenario/013_1.txt]: Maybe it should be `they'll suck up your Charm`? Think that got lost in translation slightly.

File: Cornelica/scenario/014.txt [hunk 4]
```diff
@@ -117,21 +117,21 @@
 ;; "Let's go to town hall"
 ;; "Let's consult the witch"
 
-@select s1=Let's go to town hall/*st11a s2=Let's consult the witch/*st11b
+@select s1=Let's_go_to_town_hall/*st11a s2=Let's_consult_the witch/*st11b
```
Comment (TwinDragon) [inline Cornelica/scenario/014.txt]: Missed an underscore before `witch`.

File: Cornelica/scenario/014_02b.txt [hunk 5]
```diff
@@ -4,9 +4,9 @@
 @huki b=3 id=3
 @move_f f=アルネ@笑み in=c t=150
 
-[Arne] "Welcome back! Mphm... you really like to show off!"
+[Arne]"Welcome back! Mphm... you really like to show off!"
```
Comment (TwinDragon) [inline Cornelica/scenario/014_02b.txt]: I think `Mmph` looks better for that.

File: Cornelica/scenario/014_03a.txt [hunk 6]
```diff
@@ -18,24 +18,24 @@
...
 
-[ユスティーナ]「あら、リリィちゃんじゃない。ふふ、男の子つれちゃって、デート中かしら？」 
+[Justina]"Oh, if it isn't Lily, hehe, you're with a boy, \nare you on a date?"
 
 @huki b=1 id=AC0
 @bs f=ユスティーナ@微笑 body=-1 face=0 x=350 y=50 t=100 wt=0
 @bs f=リリィ@焦り_r body=-1 face=0 x=-350 y=50 t=200
 
-[リリィ]「ア、アルはそんなんじゃないよ！」 
+[Lily]"Al isn't like that!"
```
Comment (TwinDragon) [inline Cornelica/scenario/014_03a.txt]: Referenced this way back for 013_03a I believe.

File: Cornelica/scenario/013_03a.txt (hunk 2 above)
Comment (Antielfi) [reply Cornelica/scenario/013_03a.txt]: Yep... been italian don't help with this typo

File: Cornelica/scenario/013_1.txt (hunk 3 above)
Comment (Antielfi) [reply Cornelica/scenario/013_1.txt]: The sentence means that if you get closer the charm goes up... I pharse like you will absorb charm and the magic power goes down......

File: Cornelica/scenario/014.txt (hunk 4 above)
Comment (Antielfi) [reply Cornelica/scenario/014.txt]: Fuck

File: Cornelica/scenario/014_03a.txt (hunk 6 above)
Comment (Antielfi) [reply Cornelica/scenario/014_03a.txt]: Yep, similar

File: Cornelica/scenario/014_02b.txt (hunk 5 above)
Comment (Antielfi) [reply Cornelica/scenario/014_02b.txt]: I hate those effing sound, i'm going crazy...

File: Cornelica/scenario/013_1.txt (hunk 3 above)
Comment (TwinDragon) [reply Cornelica/scenario/013_1.txt]: Maybe its just how I read it then.

**Comment by TwinDragon** [comment]
Going to close this PR now. Reference issue #22 for a couple things to review.

**Pull request closed without merging**